staging/vhost-device-gpu/src/virtio_gpu.rs

```diff
@@ -744,17 +747,25 @@ impl VirtioGpu for RutabagaVirtioGpu {
     }
 
     fn resource_assign_uuid(&self, resource_id: u32) -> VirtioGpuResult {
-        if !self.resources.contains_key(&resource_id) {
-            return Err(ErrInvalidResourceId);
-        }
+        debug_assert!(
+            self.resources.contains_key(&resource_id),
+            "Resource ID {} doesn't exists in the resources map.",
+            resource_id
+        );
```

Comment:
I don't think this should be `debug_asssert!`. I think we should always checks this and  `return Err(ErrInvalidResourceId)`.